tests/src/JIT/Regression/JitBlue/GitHub_1323/GitHub_1323.cs

```diff
@@ -0,0 +1,22 @@
+using System;
```

Comment:
Woops meant @AndreyAkinshin.  You both had such good contributions to the bug fix that I kept getting mixed up ;-)  Anyways, I'll let you know.